vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -454,11 +452,4 @@ - (void)eventLog:(const std::string &)json
   return std::make_unique<RewardsLogStream>(file, line, log_level);
 }
 
-#pragma mark - Misc
-
-- (const std::string)generateUUID
-{
-  return [self.commonOps generateUUID];
-}
-
```

Comment:
Missing method added in `vendor/brave-ios/Ads/Generated/NativeAdsClientBridge.h`:

`- (void)closeNotification:(const std::string &)id;` 

You can implement this as an empty method here but please just leave a reference to what its for/this PR